addons/l10n_ch/demo/demo_company.xml

```diff
@@ -24,7 +24,7 @@
 
     <function model="res.users" name="write">
         <value eval="[ref('base.user_root'), ref('base.user_admin'), ref('base.user_demo')]"/>
-        <value eval="{'company_ids': [(4, ref('l10n_ch.demo_company_ch'))]}"/>
+        <value eval="{'company_ids': [(4, ref('l10n_ch.demo>>_company_ch'))]}"/>
```

Comment:
🤔

Reply:
please ignore this PR, I just pushed so I would be sure to have my latest modification and be able to see them from the github interface, but I'll need to start it clean anyway :)